### ISSUE TITLE
Proptype for Context.Provider

### ISSUE THREAD

**Issue by eedrah** [facebook/prop-types#223]
Is there a proptype for Context.Provider? If I use `element` I get the following error message:

```
Warning: Failed prop type: Invalid prop `ContextProvider` of type `object` supplied to `myComponent`, expected a single ReactElement.
```

and if I use `node` I get:

```
Warning: Failed prop type: Invalid prop `ContextProvider` supplied to `myComponent`, expected a ReactNode.
```

**Comment by ljharb** [comment]
Seems related to #211.

**Comment by eedrah** [comment]
Agreed it will be fixed with #211, so closed in favor of that pull request

**Comment by henrik** [comment]
Because it took me a little while to figure out – the thing #211 ended up adding specifically is `PropTypes.elementType`, which seems to work fine for a provider.
